com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -46,6 +46,13 @@ public enum LightCookieFormat
         ColorHDR,
     }
 
+    // TODO: should this be like light cookie format, an index to a  list of formats that are tested in order for support.
+    public enum HDRFormat
+    {
+        Default = GraphicsFormat.B10G11R11_UFloatPack32,
```

Comment:
Explicit formats as enum values? Or just ints and a function to get the formats (which are of course enum ints already).

Reply:
Ok. Makes sense. I'll change it to match that.

Reply:
I changed the name to ColorBufferFormat. However, it explicitly defines the formats. 
In the past we've chosen to use abstract Low, High etc. to be more user friendly (less technical) and to be able to change the formats easier. I've kept that aspect. Any thoughts?

Reply:
Actually, this is also only for HDR and we also have SDR. I don't think it makes sense share this. I've added the ColorBuffer part to the name, since it's more exact and there can be many "HDRFormats".